Relnotes 0.17.10 (#3691)
determined-ai/determined: docs/release-notes.txt

 Version 0.17
**************

Version 0.17.10
===============

**Release Date:** March 03, 2022

**Breaking Change**

-  API: Pytorch Lightning has been updated from 1.3.5 to 1.5.9 to address a security vulnerability.
   Experiments using Pytorch Lightning Adapter with v1.3.5 are no longer supported.

**New Features**

> -  Added self-supervised image classification example in PyTorch using Bootstrap Your Own Latent

Reviewer: ```suggestion
-  Added a Bootstrap Your Own Latent (BYOL) self-supervised image classification PyTorch example.
```

   (BYOL).

-  PyTorchTrial and TFKerasTrial now automatically log the number of batches and number of records
   in every training and validation workload, as well as the duration of the workload and the
   calculated batches per second and records per second.

> -  All logs produced by tasks on the cluster are persisted and accessible through the new ``det task

Reviewer: ```suggestion
-  All (experiment & non-experiment) task logs are persisted & accessible through the new ``det task
```

   logs`` CLI command or the usual commands to retrieve their logs, even after completion.

> -  Now supports setting root certificates for the DB via the Determined Helm chart. This allows

Reviewer: ```suggestion
-  Support specifying root certificates for the DB via the Determined Helm chart. This allows
```
